frontend-runitup/src/Components/Authentication/SignUp.jsx

```diff
@@ -0,0 +1,96 @@
+import "../../styles/SignUp.css";
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+
+function SignUp() {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+  const navigate = useNavigate();
+
+  const handleChangeUser = (e) => {
+    setUsername(e.target.value);
+  };
+
+  const handleChangePassword = (e) => {
+    setPassword(e.target.value);
+  };
+
+  const handleCreate = (e) => {
+    e.preventDefault();
+
+    fetch(`${import.meta.env.VITE_BACKEND_ADDRESS}/create`, {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
```

Comment:
`headers` is repeated between signup and login. Make sure to modularize this later

Reply:
I created a file in the untils folder that contains the headers. Then pass that into the sign up and login pages.